tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -422,60 +423,80 @@ module.exports = {
         })
         .useCss()
     },
-    getFileRowButtonSelectorsByFileName: function (fileName, action) {
-      const btnSelector = this.getActionSelector(action, fileName)
-      const fileActionsBtnSelector = this.getFileActionBtnSelector(fileName)
+    /**
+     *
+     * @param {string} fileName
+     * @param {string} action
+     * @param {string} elementType
+     * @returns {{btnSelector: string, fileActionsBtnSelector: string}}
+     */
+    getFileRowButtonSelectorsByFileName: function (fileName, action, elementType = 'file') {
+      const btnSelector = this.getActionSelector(action)
```

Comment:
where is the fileName gone?

Reply:
fileName is still there. It is used while calling function getFileActionBtnSelector in line 435

Reply:
I see, the call to `getActionSelector()` was wrong before, that function only takes one argument